src/coreclr/src/tools/Common/Compiler/DependencyAnalysis/Relocation.cs

```diff
@@ -21,10 +21,11 @@ public enum RelocType
                                                     // for relative pointer (used to make NGen relocation 
                                                     // section smaller)    
         IMAGE_REL_SECREL                = 0x80,     // 32 bit offset from base of section containing target
+        IMAGE_REL_FILE_ABSOLUTE         = 0x81,     // 32 bit offset from begining of image
```

Comment:
I looked for a reloc type that can be used to emit raw offsets, relative to the beginning of the image, but none of the existing ones seemed to do exactly what I was looking for, so I added this. Please let me know if I missed something that already exists for this.

Reply:
I may be misunderstanding your exact purpose but according to the description this looks the same as <code>IMAGE_REL_BASED_ADDR32NB</code>.

Reply:
Hmm, I think I'm starting to see your point - if you need a relocation against file positions, not mapped RVA's, then you're right that this is not covered by the pre-existing relocation types.